src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,36 @@
+.container {
+  @include center-flex;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
```

Comment:
according BEM it's not allowed to position your block, only your .container (father of .stopwatch) can set the position